yarn.lock

```diff
@@ -3555,6 +3555,11 @@ minipass@^3.0.0:
   dependencies:
     yallist "^4.0.0"
 
+minipass@^5.0.0:
```

Comment:
just curious why is it required?

Reply:
Here's the `yarn why` so `node-pre-gyp` needs it.

```bash
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "minipass"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "minipass@3.1.6"
info Reasons this module exists
   - "@mapbox#node-pre-gyp#tar" depends on it
   - Hoisted from "@mapbox#node-pre-gyp#tar#minipass"
   - Hoisted from "@mapbox#node-pre-gyp#tar#fs-minipass#minipass"
   - Hoisted from "@mapbox#node-pre-gyp#tar#minizlib#minipass"
```